BSB_lan_defs.h

```diff
@@ -7678,9 +7773,12 @@ PROGMEM_LATE const cmd_t cmdtbl2[]={
 {0x593D1769,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8430,  STR8430,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Quelle Austritt Min
 {0x053D195A,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8431,  STR8431,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Quellenzw'kreis Vorlauftemp
 {0x053D196D,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8434,  STR8434,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Sauggastemperatur
-{0x053D163B,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8435,  STR8435,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Verdampfungstemperatur
-{0x593D1662,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8436,  STR8436,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Überhitzung oder Überhitzungssollwert
-// vorheriges könnte auch 1663 sein weil da noch Überhitzungssollwert übermittelt wird
+{0x593D1664,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8435,  STR8435,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Verdampfungstemperatur
+// todo: How to integrate this line? it's sent along with parameter 8435
+// {0x053D163B,  CAT_DIAG_ERZEUGER,    VT_PRESSURE_WORD, ????,  STR8435_1, 0,                   NULL,         FL_RONLY,     DEV_ALL}, // Verdampfungsdruck
+{0x593D1663,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8436,  STR8436,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Überhitzung
+// todo: How to integrate this line? it's sent along with parameter 8436
+// {0x593D1662,  CAT_DIAG_ERZEUGER,    VT_TEMP,          ????,  STR8436_1, 0,                   NULL,         FL_RONLY,     DEV_ALL}, // Überhitzungssollwert
```

Comment:
Not sure which parameter number to assign. The datagram is sent and displayed along with parameter 8436.

Reply:
8439 would still be free, but first we have to figure out the conflicts between your observations regarding the CommandIDs and the parameter numbers and the ones from the other user who has reported these CommandIDs two years ago (for device family 108). It could be that these are used differently on different systems, but since they are so similar, I assume that there is a different reason for that...

Reply:
Ok, the intial comment for 8436
> // Überhitzung oder Überhitzungssollwert
was an indication for me that the other user was actually not sure if the observations were correct (don't want to offend anyone). That's why I changed the lines and split them into Überhitzung and Überhitzungssollwert respectivly. Would be cool if someone could double check these and the other command ids as I agree with you that the differences are strange and I barely cannot imagine that a manufacturer like Siemens is doing such weird things😀

Reply:
I contacted the other user and he said he would be looking into it over the weekend and compare with your contributions. Back then he added almost all Fujitsu-specific parameters, so could well be that among those many some of them got mixed up.

Reply:
Did you get any news from the other user?

Reply:
So sorry for the delayed response, I'm really loaded with work at the moment. Yes, the user we THOUGHT had added these parameters replied, but it turned out it wasn't him, so we're a bit clueless where these parameters came from. I will have to look closer at the submissions again to find out when these have been added...

Reply:
No problem, thanks for the update.